spec/models/validations/sales/sale_information_validations_spec.rb

```diff
@@ -68,17 +70,29 @@
         end
       end
 
-      context "and form year is 2024 or later" do
-        let(:record) { build(:sales_log, hodate: Date.new(2021, 12, 1), saledate: Date.new(2024, 12, 1)) }
+      context "and form year is 2025 or later" do
+        let(:record) { build(:sales_log, hodate: current_collection_start_date - 3.years, saledate: current_collection_start_date) }
 
-        it "adds an error" do
+        it "does not add an error" do
           sale_information_validator.validate_practical_completion_date(record)
 
-          expect(record.errors[:hodate]).to be_present
+          expect(record.errors[:hodate]).to be_empty
+          expect(record.errors[:saledate]).to be_empty
         end
       end
     end
 
+    context "when hodate 5 or more years before saledate" do
+      let(:record) { build(:sales_log, hodate: current_collection_start_date - 5.years, saledate: current_collection_start_date) }
```

Comment:
there's some calcs going on with the dates here, wonder if this is the most resilient way?